tensorflow/core/api_def/base_api/api_def_ParallelInterleaveDatasetV2.pbtxt

```diff
@@ -81,13 +71,8 @@ The `tf.data` Python API creates instances of this op from
 `Dataset.interleave()` when the `num_parallel_calls` parameter of that method
 is set to any value other than `None`.
 
-!! WARNING !! If the `sloppy` parameter is set to `True`, the operation of this
-dataset will not be deterministic!
-
-The `sloppy` parameter of this op always starts out set to `False` when the
-Python API creates instances of this op. If the `make_sloppy` static
-optimization pass runs, that optimization will set the `sloppy` parameter to
-`True`. Whether this optimization is performed is determined by the
-`experimental_deterministic` parameter of `tf.data.Options`.
+If the `experimental_deterministic` parameter of `tf.data.Options` is set to
```

Comment:
As the default of that argument is `True`, I would phrase this the other way around, pointing out that in order to avoid head of line blocking, one can set the `experimental_deterministic` parameter to `False` (which will can improve performance at the expense of non-determinism).

Reply:
Thanks for pointing that out! I've updated the documentation for deterministic behavior of this op with a more complete explanation of the `experimental_deterministic` parameter in commit 9dbd330.